test/spatial/ABC.jl

```diff
@@ -24,6 +24,9 @@ netstoch = [[1 => -1, 2 => -1, 3 => 1],[1 => 1, 2 => 1, 3 => -1]]
 rates = [0.1/mesh_size, 1.]
 majumps = MassActionJump(rates, reactstoch, netstoch)
 
+rx_coefficients = ones(get_num_majumps(majumps), num_nodes)
```

Comment:
Can we get test coverage for the different mass action jump variants, constant in space via a vector, non-constant in space via a matrix, and a hyrbid of the two?

It would be good to have one accuracy test for each case to ensure no bugs.